src/lib/formats/ap2_dsk.cpp

```diff
@@ -542,37 +544,39 @@ bool a2_16sect_format::save(util::random_read_write &io, const std::vector<uint3
 							// but only write it if the bitfield of the track shows datagood is NOT set.
 							// if it is set we don't want to overwrite a guaranteed good read with a bad one
 							// if past read had a bad checksum or bad postamble...
-#ifndef USE_OLD_BEST_SECTOR_PRIORITY
-							if (((visualgrid[se][track]&DATAGOOD)==0)||((visualgrid[se][track]&DATAPOST)==0)) {
-								// if the current read is good, and postamble is good, write it in, no matter what.
-								// if the current read is good and the current postamble is bad, write it in unless the postamble was good before
-								// if the current read is bad and the current postamble is good and the previous read had neither good, write it in
-								// if the current read isn't good and neither is the postamble but nothing better
-								// has been written before, write it anyway.
-								if ( ((data[0x156] == c) && (dpost&0xFFFF00)==0xDEAA00) ||
-													(((data[0x156] == c) && (dpost&0xFFFF00)!=0xDEAA00) && ((visualgrid[se][track]&DATAPOST)==0)) ||
-													(((data[0x156] != c) && (dpost&0xFFFF00)==0xDEAA00) && (((visualgrid[se][track]&DATAGOOD)==0)&&(visualgrid[se][track]&DATAPOST)==0)) ||
-													(((data[0x156] != c) && (dpost&0xFFFF00)!=0xDEAA00) && (((visualgrid[se][track]&DATAGOOD)==0)&&(visualgrid[se][track]&DATAPOST)==0))
-													) {
+							if constexpr (USE_OLD_BEST_SECTOR_PRIORITY) {
+								if ((visualgrid[se][track]&DATAGOOD)==0) {
 									for(int i=0x56; i<0x156; i++) {
 										uint8_t dv = data[i];
 										*dest++ = dv;
 									}
 								}
 							}
-#else
-							if ((visualgrid[se][track]&DATAGOOD)==0) {
-								for(int i=0x56; i<0x156; i++) {
-									uint8_t dv = data[i];
-									*dest++ = dv;
+							else {
```

Comment:
This file is already inconsistent with K&R vs Allman formatting, but I don’t think any of it uses this style:
```c++
    }
    else {
```

It’s all either Allman:
```c++
    }
    else
    {
```
or K&R:
```c++
    } else {
```

You should use the predominant style of the file.

Reply:
FWIW, this style is used in this function too (see the block starting with `if(v == 0xff) {`). But sure, I changed it.